application_form/api/views.py

```diff
@@ -40,3 +41,25 @@ def get(self, request, project_uuid):
         )
         serializer = self.get_serializer(reservations, many=True)
         return Response(serializer.data)
+
+
+class LatestApplicantInfo(GenericAPIView):
+    """
+    Public: Returns the primary applicant from the latest application.
+    """
+
+    serializer_class = ApplicantSerializer
+    http_method_names = ["get"]
+
+    def get(self, request, customer_id):
+        try:
+            application = Application.objects.filter(customer__id=customer_id).latest(
+                "created_at"
+            )
+            applicant = application.applicants.filter(is_primary_applicant=True).first()
+            if applicant:
+                serializer = self.get_serializer(applicant)
+                return Response(serializer.data)
+            return Response({"message": "No primary applicant found"}, status=404)
```

Comment:
Actually we are trying to add more errors to sentry because at the moment it is hard to debug since nothing much goes to sentry and we have to dig in kibana. But what is your suggestion here?

Reply:
`raise APIException("No primary applicant found")` will result in 500 and should create a sentry report.